config/externalname.go

```diff
@@ -1051,6 +1051,8 @@ var terraformPluginSDKExternalNameConfigs = map[string]config.ExternalName{
 	"google_bigquery_analytics_hub_listing": config.TemplatedStringAsIdentifier("listing_id", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.location }}/dataExchanges/{{ .parameters.data_exchange_id }}/listings/{{ .external_name }}"),
 	// Imported by using the following projects/{{project}}/locations/{{location}}/dataExchanges/{{data_exchange_id}} roles/viewer user:jane@example.com
 	"google_bigquery_analytics_hub_data_exchange_iam_member": config.IdentifierFromProvider,
+	// Imported by using the following projects/{{project}}/locations/{{location}}/subscriptions/{{subscription_id}}
+	"google_bigquery_analytics_hub_listing_subscription": config.TemplatedStringAsIdentifier("subscription_id", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.location }}/subscriptions/{{ .external_name }}"),
```

Comment:
The resource ID `(projects/{{project}}/locations/{{location}}/subscriptions/{{subscription_id}})` includes a provider-generated `subscription_id` that isn’t user-defined or configurable in Terraform.

Since the identifier is fully assigned by the provider, this should use: `config.IdentifierFromProvider`

Using `config.TemplatedStringAsIdentifier` here isn’t appropriate because there’s no parameter or argument in the schema corresponding to `subscription_id`. The provider handles this ID creation internally.